lldb/test/API/functionalities/tsan/swift-access-race/TestTsanSwiftAccessRace.py

```diff
@@ -54,22 +54,25 @@ def do_test(self):
                 runtimes.append(os.path.join(libspec.GetDirectory(), libspec.GetFilename()))
         self.registerSharedLibrariesWithTarget(target, runtimes)
 
-        self.runCmd("run")
-
-        stop_reason = self.dbg.GetSelectedTarget().process.GetSelectedThread().GetStopReason()
-        if stop_reason == lldb.eStopReasonExec:
-            # On OS X 10.10 and older, we need to re-exec to enable
-            # interceptors.
-            self.runCmd("continue")
+        # Unfortunatley the runtime itself isn't 100% reliable in reporting TSAN errors.
+        process = None
+        stop_reason = lldb.eStopReasonInvalid
+        for retry in range(5):
+            process = target.LaunchSimple(None, None, self.get_process_working_directory())
```

Comment:
I don't understand how not being reliable in reporting TSAN errors would cause target.LaunchSimple to fail to produce a process.  I would have expected the process to run to completion and then you have a process in eStateExited.  What's actually happening here?